config/initializers/okcomputer.rb

```diff
@@ -18,6 +18,15 @@ def version
 end
 OkComputer::Registry.register 'version', VersionCheck.new
 
+# Pings a specific solr core -- assumes Solr is configured to use PingRequestHandler
+# @see https://cwiki.apache.org/confluence/display/solr/Ping
+class SolrCoreCheck < OkComputer::HttpCheck
```

Comment:
Host is a misnomer; it'll happily take a path to a core, e.g. https://github.com/sul-dlss/exhibits/blob/master/config/initializers/okcomputer.rb#L7